editing_and_imputation/editing/totals_components/technical_specification.md

```diff
@@ -0,0 +1,291 @@
+
+```
+    x_{absolute} <= |y_{total, t} - y_{total, predictive}|
+```
+
+If this is true we mark 
+
+```
+    satisfied = TRUE
+``` 
+
+else we mark 
+
+```
+    satisfied = FALSE
+```
+
+we return the satisfied boolean and go to stage 6 in section 5.5 (if boolean is ```TRUE```), if boolean is ```FALSE``` we move to section 5.4.3.
+
+Note: Before we leave this stage this point we need to check that zero error condition 3 is satisfied
+
+```
+    y_{total, t} = 0
+```
+
+,
+
+```
+y_{derived, t} > 0
+```
+
+and amend total = ```True```.
+
+If so apply the correction to override the components with zeros if the difference observed is within the tolerances determined by the detection method.
+
+We then set TCC = M for manual checking.
+### 5.4.2 Check Percentage Difference Threshold (Stage 5b)
+
+If the total for the predictive period is within the low and high percentage as shown by the formula below 
+
+```
+    y_{derived} - (y_{derived} * x_{percent}) =< y_{total, predictive} =< y_{derived} + (y_{derived} * x_{percent})
+```
+
+If this is true we mark 
+
+```
+    satisfied = TRUE
+``` 
+
+else we mark 
+
+```
+    satisfied = FALSE
+```
+
+we return the satisfied boolean and go to stage 6 in section 5.5 (if boolean is ```TRUE```), if boolean is ```FALSE``` we return a TCC = M marker meaning manual editing is required and write an output.
+
+### 5.4.3 Percentage Difference Threshold
+
+This stage is to check if 
+
+```
+    x_{percent} ≠ None
+```
+
+If this is true we run stage 5b (see section 5.4.2 above), else we check that
+
+```
+    x_{absolute} ≠ None
+```
+or
+
+```
+    x_{percent} ≠ None
+```
+
+If it is true we return a TCC = M marker meaning manual editing is required and we write an output.
+
+### 5.5 Error Correction (Stage 6)
+
+This section covers the error correction aspect of the process.
+
+At this point we rely on the amend value `do_amend_total`. This is where `y_derived` or `y_total` are > 0.
+
+If 
+
+```
+    do_amend_total = TRUE
+```
+
+Then the total is automatically corrected for the target period to equal the sum of the components in the target period.
+
+Hence we set the following
+
+```
+    y_{derived} = y_{total, t}
+```
+
+and the final values for all the components match their originals. 
+
+We would then have a TCC = T marker and write the output.
+
+However, if
+
+```
+    do_amend_total = FALSE
+```
+
+Then the components are automatically corrected for the target period to equal the total in the target period.
+
+```
+    y_{c} = (\frac{y_{c} + \dots + y_{derived}}) * y_{total, predictive}
+```
+
+and the final values for all the components match their originals. 
+
+If any of the components are not corrected when needed then we repeat the formula above. Else we set
+
+```
+    y_{total, t} = y_{total, predictive}
+```
+
+and return TCC = C marker and write the output.
+
+If we have the situation as described above we will need to check the fourth zero case. This can be achieved by checking if
```

Comment:
I don't think you need to capture much more than say, in the case where the predictive total is set to zero and the amend_total indicates that the components need to be adjusted this step of the method will ensure that each component is reset to zero to match the expected total.